src/Generators/DataTablesMakeCommand.php

```diff
@@ -101,7 +177,10 @@ protected function replaceModelImport(&$stub)
      */
     protected function getStub()
     {
-        return __DIR__ . '/stubs/datatables.stub';
+        $config = $this->laravel['config'];
+        return $config->get('datatables.custom_template')
```

Comment:
I think we should use the buttons config namespace and maybe set the key to `stub` for clarity?

```php
return $config->get('datatables-buttons.stub')
```